README.md

```diff
@@ -652,3 +652,10 @@ See [Settings](#settings) above for examples and to understand what you can conf
 The server and client are the user-facing components of the API system, but there are a number of other components that work together to make sure your settings are applied, and that they survive upgrades of Bottlerocket.
 
 For more details, see the [API system documentation](sources/api/).
+
+### Default Volumes
+
+Bottlerocket operates with two default storage volumes. 
+* The root device, `/dev/xvda`, holds the active and passive [partition sets](#updates).
```

Comment:
```suggestion
* The root device, `/dev/xvda`, holds the active and passive [partition sets](#updates-1).
```
Sorry, I should have tested this link suggestion.  There are multiple sections named "updates" and I intended to point to the one shortly above this section that discusses partition sets.